java-samples/corda5-negotiation-cordapp/build.gradle

```diff
@@ -46,12 +53,45 @@ allprojects {
         // All dependencies are held in Maven Central
         mavenLocal()
         mavenCentral()
+        // Remove once Corda 5.1 and Flow Testing Driver are full GA
+        maven {
+            url = "$artifactoryContextUrl/corda-dependencies-dev"
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = findProperty('cordaArtifactoryUsername') ?: System.getenv('CORDA_ARTIFACTORY_USERNAME')
+                password = findProperty('cordaArtifactoryPassword') ?: System.getenv('CORDA_ARTIFACTORY_PASSWORD')
+            }
+            mavenContent {
+                snapshotsOnly()
+            }
+        }
+        // Remove once Corda 5.1 and Flow Testing Driver are full GA
+        maven {
+            url = "$artifactoryContextUrl/corda-os-maven"
+            authentication {
+                basic(BasicAuthentication)
+            }
+            credentials {
+                username = findProperty('cordaArtifactoryUsername') ?: System.getenv('CORDA_ARTIFACTORY_USERNAME')
+                password = findProperty('cordaArtifactoryPassword') ?: System.getenv('CORDA_ARTIFACTORY_PASSWORD')
+            }
+        }
     }
 
     tasks.withType(Test).configureEach {
         useJUnitPlatform()
+        doFirst {
+            systemProperty 'org.slf4j.simpleLogger.defaultLogLevel', 'info'
+            systemProperty 'org.slf4j.simpleLogger.dateTimeFormat', 'yyyy-MM-dd HH:mm:ss:SSS Z'
+            systemProperty 'org.slf4j.simpleLogger.showDateTime', true
+            systemProperty 'org.slf4j.simpleLogger.showShortLogName', true
+            systemProperty 'org.slf4j.simpleLogger.showThreadName', false
+            systemProperty 'org.slf4j.simpleLogger.logFile', 'System.out'
+            systemProperty 'org.slf4j.simpleLogger.log.org.apache.aries.spifly.BaseActivator', 'OFF'
```

Comment:
You've got this in both the root build.gradle and the workflows/build.gradle
As it's only for the driver tests, do you want to remove it from here?

Reply:
Removed.